src/adagrams.js

```diff
@@ -1,8 +1,169 @@
-const Adagrams = {
-  drawLetters() {
-    // Implement this method for wave 1
-  },
+class Adagrams {
+  static frequencies() {
+    return [
+    ["A",9],
+    ["B", 2],
+    ["C", 2],
+    ["D", 4],
+    ["E", 12],
+    ["F", 2],
+    ["G", 3],
+    ["H", 2],
+    ["I", 9],
+    ["J", 1],
+    ["K", 1],
+    ["L", 4],
+    ["M", 2],
+    ["N", 6],
+    ["O", 8],
+    ["P", 2],
+    ["Q", 1],
+    ["R", 6],
+    ["S", 4],
+    ["T", 6],
+    ["U", 4],
+    ["V", 2],
+    ["W", 2],
+    ["X", 1],
+    ["Y", 2],
+    ["Z", 1],
+  ];}
+
+  static makeNewBag() {
+    let bag = [];
```

Comment:
This is a very cool weird thing about JavaScript: You can make `bag` a `const`. Even though you eventually call `bag.push` to modify `bag`, you never actually re-assign it, so JS says that `const` is fine. :)

There are a couple more opportunities in your project where you could go from `let` to `const`-- feel free to play around and rely on your tests to see if it works